providers/amazon/src/airflow/providers/amazon/aws/operators/sagemaker_unified_studio.py

```diff
@@ -0,0 +1,148 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""This module contains the Amazon SageMaker Unified Studio Notebook operator."""
+
+from functools import cached_property
+
+from airflow import AirflowException
+from airflow.configuration import conf
+from airflow.models import BaseOperator
+from airflow.providers.amazon.aws.hooks.sagemaker_unified_studio import (
+    SageMakerNotebookHook,
+)
+from airflow.providers.amazon.aws.triggers.sagemaker_unified_studio import (
+    SageMakerNotebookJobTrigger,
+)
+from airflow.utils.context import Context
+
+
+class SageMakerNotebookOperator(BaseOperator):
```

Comment:
I'm missing handling of operator extra link here.

Reply:
Sorry, could you clarify? What's the operator extra link?

Reply:
We don't have great docs on extra links to describe what they are (we should fix that). It's basically a way to add hyperlinks to the details of a given task. It's common to link to the cloudwatch logs that pertain to the aws operator in question, or link to the console page where the job/task itself can be viewed. You can search around in our other operators to see examples and get an idea of how it works.

Reply:
https://airflow.apache.org/docs/apache-airflow/stable/howto/define-extra-link.html#define-an-operator-extra-link

Reply:
Added a `SageMakerUnifiedStudioLink` in the latest commit